src/PKSim.Core/Model/IndividualSimulation.cs

```diff
@@ -1,69 +1,26 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using OSPSuite.Utility.Collections;
 using OSPSuite.Utility.Extensions;
 using OSPSuite.Utility.Visitor;
 using OSPSuite.Core.Domain;
 using OSPSuite.Core.Domain.Data;
 using OSPSuite.Core.Domain.Services;
 using PKSim.Core.Chart;
+using PKSim.Core.Model.Extensions;
 
 namespace PKSim.Core.Model
 {
    public class IndividualSimulation : Simulation
    {
       private DataRepository _dataRepository;
-      private readonly ICache<string, CompoundPK> _compoundPKCache;
 
-      public IndividualSimulation()
+      public IndividualSimulation() : base()
       {
-         _compoundPKCache = new Cache<string, CompoundPK>(x => x.CompoundName, x => new CompoundPK());
       }
 
       public virtual IEnumerable<SimulationTimeProfileChart> TimeProfileAnalyses => Analyses.OfType<SimulationTimeProfileChart>();
 
-      public void ClearPKCache()
```

Comment:
@abdelr It would be good to comment a bit on PR. I have no idea why this is removed. This has nothing to do with pk properties for simulations

Reply:
It was moved to the base class so population simulation can also use it.